ReactNativeClient/lib/models/Setting.js

```diff
@@ -36,7 +36,22 @@ class Setting extends BaseModel {
 				type: Setting.TYPE_STRING,
 				public: false,
 			},
-
+			'keyboardMode': {
+				value: 'default',
+				type: Setting.TYPE_STRING,
+				public: true,
+				appTypes: ['desktop'],
+				isEnum: true,
+				label: () => _('Keyboard Mode'),
+				section: 'appearance',
```

Comment:
Let's leave it in the General section for now (so you can simply remove the `section` key). Eventually, we might want to add an Editor section.